docs/source/about/guides/linux/headless_ssh.rst

```diff
@@ -0,0 +1,481 @@
+       | 30%   46C    P2              45W / 220W |    549MiB /  8192MiB |      2%      Default |
+       |                                         |                      |                  N/A |
+       +-----------------------------------------+----------------------+----------------------+
+
+       +---------------------------------------------------------------------------------------+
+       | Processes:                                                                            |
+       |  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
+       |        ID   ID                                                             Usage      |
+       |=======================================================================================|
+       |    0   N/A  N/A      1393      G   /usr/lib/Xorg                                86MiB |
+       |    0   N/A  N/A      1440    C+G   sunshine                                    293MiB |
+       +---------------------------------------------------------------------------------------+
+
+#. Check ``/dev/uinput`` permissions
+
+   .. code-block:: bash
+
+      ls -l /dev/uinput
+
+   .. code-block:: console
+
+      crw------- 1 <user> <primary_group> 10, 223 Aug 29 17:31 /dev/uinput
+
+#. Connect to Sunshine host from a moonlight client
+
+*Now unplug your monitors and repeat steps 1 - 5*
+
+
+SSH Client Script (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At this point you have a working setup! For convience I created this bash script to automate the
+startup of the X server and Sunshine on the host.
+This can be run on Unix systems, or on Windows using the ``git-bash`` or any bash shell.
+
+For Android/iOS you can install Linux emulators, e.g. ``Userland`` for Android and ``ISH`` for iOS.
+The neat part is that you can execute one script to launch Sunshine from your phone or tablet!
+
+.. code-block:: bash
+
+   #!/bin/bash
+
+   ssh_args="<user>@192.168.X.X" # Or use alias set in ~/.ssh/config
+
+   check_ssh(){
+     result=1
+      # Note this checks infinitely, you could update this to have a max # of retries
+     while [[ $result -ne 0 ]]
+     do
+       echo "checking host..."
+       ssh $ssh_args "exit 0" 2>/dev/null
+       result=$?
+       [[ $result -ne 0 ]] && {
+          echo "Failed to ssh to $ssh_args, with exit code $result"
+       }
+       sleep 3
+     done
+     echo "Host is ready for streaming!"
+   }
+
+   start_stream(){
+     echo "Starting sunshine server on host..."
+     echo "Start moonlight on your client of choice"
+      # -f runs ssh in the background
+     ssh -f $ssh_args "~/scripts/sunshine.sh &"
+   }
+
+   check_ssh
+   start_stream
+   exit_code=${?}
+
+   sleep 3
+   exit ${exit_code}
+
+Done
+----
+
+Congrats you can now stream your desktop headless! When trying this the first time,
+keep your monitors close by incase something isn't working right.
+
```

Comment:
Add some next steps,

reference: https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/app_examples.html#changing-resolution-and-refresh-rate-linux-x11

for setting resolution on Sunshine host to match moonlight client device resolution.